### DISCUSSION TITLE
What is the folder `.swc` generated in Next.js 12.2

### DISCUSSION THREAD

**Discussion by ixartz** [vercel/next.js#38270] [Help] [answered]
Hi,

I just updated my project to Next.js 12.2 and I notice Next.js automatically create a folder named `.swc`.
This folder contains another named `plugins`and `plugins` contains `v2`.
So here is the structure: `.swc` > `plugins` > `v2` but `v2` folder is empty.

Is it normal? Why is it empty?

Should I .swc as an ignored files in git? Should I track on Git?

**Comment by icyJoseph** [comment]
@kdy1 Sorry to bother you on the weekend, but I think you know a lot more about SWC than I do. I am also seeing this `.swc` folder, but it is empty through, so git ignores it as of now.

**Comment by kwonoj** [comment, marked as answer]
This should be resolved via https://github.com/vercel/next.js/pull/38175 .
